### ISSUE TITLE
Does Kinect V2 works in OSX ?

### ISSUE THREAD

**Issue by omarojo** [bradenneufeld/K2P3Toolkit#4]
I wonder if the Kinect v2 works in OSX with the drivers mentioned. 
I know the very first Kinect version fox Xbox works in OSX but I thought the new one would only work with on Windows with their SDK.

**Comment by bnfld** [comment]
Yes, these Kinect v2 drivers can still be used with macOS. According to @shiffman [you need 10.9 or higher](https://github.com/shiffman/OpenKinect-for-Processing) - we have used his build of the libfreenect2 drivers for this fork.
